### DISCUSSION TITLE
next/image default loader: generate images only once on first request

### DISCUSSION THREAD

**Discussion by stefanprobst** [vercel/next.js#19231] [Help]
i am serving images from the `public` folder with `next/image`'s default loader.

if i am understanding [the docs](https://nextjs.org/docs/basic-features/image-optimization#caching) correctly, currently these images will be generated on request and cached for 60s, but then deleted and regenerated again when they are requested after that.

is it possible to only generate images once, the first time they are requested, but then cache them indefinitely (or with a much higher ttl)?

**Comment by HassanMuawiya313** [comment]
Pleas read first documentation before playing images

**Reply by stefanprobst** [reply]
please point me to where the docs answer my question. thanks
